th2_data_services/provider/v5/commands/http.py

```diff
@@ -11,23 +11,273 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+from functools import partial
 from typing import List
 
+from th2_data_services.provider.v5.adapters.basic_adapters import AdapterSSE
+from th2_data_services.data import Data
 from th2_data_services.provider.v5.command import IHTTPProvider5Command
 from th2_data_services.provider.v5.data_source.http import HTTPProvider5DataSource
+from th2_data_services.provider.v5.provider_api import HTTPProvider5API
+
+import logging
+
+logger = logging.getLogger("th2_data_services")
+logger.setLevel(logging.DEBUG)
 
 
 class GetEventById(IHTTPProvider5Command):
+    """A Class-Command for request to rpt-data-provider.
+
+    It retrieves the event by id.
+
+    Returns:
+        dict: Th2 event.
+    """
+
     def __init__(self, id: str):
+        """
+        Args:
+            id: Event id.
+
+        """
         self._id = id
 
-    def handle(self, data_source: HTTPProvider5DataSource):
-        pass
+    def handle(self, data_source: HTTPProvider5DataSource) -> dict:
+        api: HTTPProvider5API = data_source.source_api
+        url = api.get_url_find_event_by_id(self._id)
+
+        logger.info(url)
+
+        return api.execute_request(url).json()
 
 
 class GetEventsById(IHTTPProvider5Command):
+    """A Class-Command for request to rpt-data-provider.
+
+    It retrieves the events by ids.
+
+    Returns:
+        List[dict]: Th2 events.
+    """
+
+    def __init__(self, ids: List[str]):
+        """
+        Args:
+            ids: Event id list.
+
+        """
+        self._ids: ids = ids
+
+    def handle(self, data_source: HTTPProvider5DataSource):
+        api: HTTPProvider5API = data_source.source_api
+        url = api.get_url_find_events_by_id(*self._ids)
```

Comment:
We do not should use `get_url_find_events_by_id` because it is deprecated.
Also, it has a problem - we cannot get more than 64 events per one request (there was a bug).

Use `GetEventById` instead.
We also have to add `use_stub` method and `apply_adapter`. Please see https://exactpro.atlassian.net/browse/TH2-2986 and grpc commands.